src/web_app_skeleton/Procfile.dev.ecr

```diff
@@ -1,3 +1,4 @@
+system_check: ./script/system_check
```

Comment:
Run it for both API and web because you may have external deps in both cases.

Reply:
I *think* you can script/system_check without the `./`

Reply:
Also thoughts on `check_system` so it is more of a verb? Like "Hey computer, check my system". Kind of like if we had system in `setup`, setup_system` would read better than `system_setup` IMO

What do you think

Reply:
So `check_system: script/check_system`? or `system_check: script/check_system` like the first part is the noun and the second part is the verb? @edwardloveall ^?

Reply:
Yeah, I was thinking `check_system: script/check_system` but would love to hear what Edward thinks too :)

Reply:
I think I'm partial to check system? but honestly this might be bikeshedding too much. I think maybe we're struggling because they're both clear. The only way they could be more clear is if you added an article like `check_the_system` or a subject like `system_check_yourself_before_you_wreck_yourself`. I have no idea if that's correct and I'm mostly making this up. I think you could keep it as-is and we can switch it if it becomes "wrong" later.

Reply:
Oh, I think you are correct. You definitely have to check yourself because wrecking yourself is a very real thing. 

I'll roll with `check_system`. After saying a few times, I like it better.